labdao/plex
# Experiment listing moved to side nav

## What type of PR is this? 

- 🎮 Feature
- 🍧 Refactor

## Description

This PR updates the side `Nav` component to dynamically display a user's Experiments, similar to a Chat-GPT interface. Previous features related to Experiment panel views were removed to allow for predictable click-routing behavior.

Time categories for Experiments are as follows
1. Today
2. Previous 7 Days
3. Previous 30 Days
4. Older

## Steps to Test

1. Run a number of labsay experiments on sped-up mode
2. Update the timestamps manually in the DB
3. Expected behavior is to see User Experiments loaded in the side nav, with categories based on timestamps from DB fetch
4. Click a few of the experiment names to navigate to full page experiment detail view. Expected behavior is to not see the panel view

<img width="1441" alt="Screenshot 2024-03-07 at 10 58 06 AM" src="https://github.com/labdao/plex/assets/29086101/ca73045f-bca5-43ab-8eac-f5b185758e1b">
